clients/src/main/java/org/apache/kafka/clients/consumer/internals/PrototypeAsyncConsumer.java

```diff
@@ -547,7 +642,7 @@ public ConsumerGroupMetadata groupMetadata() {
 
     @Override
     public void enforceRebalance() {
-        throw new KafkaException("method not implemented");
+        enforceRebalance(null);
```

Comment:
This ends up throwing "Not implemented" exception. Should we throw something like "Not supported" instead? We will never implement this with the new protocol (I'm ok with leaving this call here, just referring to the exception message)

Reply:
I'm just going to change it back to it's original form:

```java
@Override
public void enforceRebalance() {
    throw new KafkaException("method not implemented");
}
```

Reply:
Reverted `enforceRebalance()`.